packages/dev/test/RelayServer.test.ts

```diff
@@ -605,6 +606,212 @@ contract('RelayServer', function (accounts: Truffle.Accounts) {
     })
   })
 
+  describe('withdrawToOwnerIfNeeded', function () {
+    let configFilename: string
+    let currentBlockNumber: number
+    const onceWithdrawConfig = JSON.stringify({
+      withdrawOnEthBalanceReached: 1e18,
+      leaveManagerWithAmountEth: 0.5e18,
+      repeat: false
+    })
+    beforeEach(async function () {
+      await env.newServerInstance({}, getTemporaryWorkdirs())
+      configFilename = `${env.relayServer.config.workdir}/withdraw.json`
+      assert.isFalse(fs.existsSync(configFilename))
+      currentBlockNumber = await env.web3.eth.getBlockNumber()
+    })
+    afterEach(async function () {
+      sinon.restore()
+    })
+    it('should not withdraw if relayer is not ready', async function () {
```

Comment:
The problem that I see with it is that we don't really know why it's not ready (which is a known design issue with the server...) so I cannot tell whether it's not ready because it's not connected to the node, the DB is corrupted etc. So I don't know that in all cases where the server isn't ready, it's still ok to try to withdraw. What do you think?

Reply:
Now that I added a cli to withdraw, there is no need to allow withdrawal through the relay process when it's not ready.

Reply:
is the test still valid?

Reply:
Yes, the relayer should now withdraw when it's not ready. If you really want to withdraw now, use the cli.